src/store/actions/teams.js

```diff
@@ -0,0 +1,83 @@
+import { Toast } from '../../config/Notifications';
+import * as actionTypes from './actionTypes';
+
+import { getTeams, createTeam } from '../../services/teamService';
+
+const _fetchAllTeams = () => {
+  return {
+    type: actionTypes.FETCHING_TEAMS,
+  };
+};
+
+const _fetchAllTeamsFailed = error => {
+  return {
+    type: actionTypes.FETCHING_TEAMS_FAILURE,
+    error,
+    resultCount: 0,
+  };
+};
+
+const _receiveAllTeams = (teams, count) => {
+  return {
+    type: actionTypes.FETCHING_TEAMS_SUCCESS,
+    teams,
+    resultCount: count,
+  };
+};
+
+const _postNewTeam = () => {
+  return {
+    type: actionTypes.CREATING_NEW_TEAM,
+  };
+};
+
+const _postNewTeamFailed = error => {
+  return {
+    type: actionTypes.CREATING_NEW_TEAM_FAILURE,
+    error,
+  };
+};
+
+const _postNewTeamSuccess = (team) => {
+  return {
+    type: actionTypes.CREATING_NEW_TEAM_SUCCESS,
+    team,
+  };
+};
+
+export const updateSelectedTeamId = (selectedTeamId) => {
```

Comment:
brackets

Reply:
Have these changes been linted @paulmatchett? I would have thought we would have had a rule to enforce a consistent bracket style

Reply:
Bracket removed. I think the lint file must be quite light as there seems to be quite a lot of rules not applying. I'll look into this.

Reply:
This is the rule you're looking for @paul-matchett  https://eslint.org/docs/rules/arrow-parens

Reply:
Thanks @girwin575. This should be added as part of a new ticket going forward.

Reply:
Code style should be handled by something like Prettier rather than ES-Lint. Check out Dan Abramov's blog post that covers some of this stuff:
https://overreacted.io/writing-resilient-components/

Reply:
Hi @William-Unosquare I'm pretty certain I was told not to use Prettier. Someone definitely asked me to make sure it wasn't being used when I first started having these issues?